src/tab/types.ts

```diff
@@ -1,4 +1,6 @@
-export interface TabProps {
+import { TabLinkChainBaseProps } from "./tab-link-chain";
+
+export interface TabProps extends TabLinkChainBaseProps {
```

Comment:
could TabLinkChain Props `Pick` specific props from `TabProps` instead?

as this is a public types file, letting it extend an internal type feels weird

Reply:
will derive from `tab.item`